flight/Modules/UAVOMSPBridge/UAVOMSPBridge.c

```diff
@@ -132,13 +133,22 @@ typedef enum {
 	MSP_MAYBE_UAVTALK2,
 	MSP_MAYBE_UAVTALK3,
 	MSP_MAYBE_UAVTALK4,
-	MSP_MAYBE_UAVTALK_SLOW2,
-	MSP_MAYBE_UAVTALK_SLOW3,
-	MSP_MAYBE_UAVTALK_SLOW4,
-	MSP_MAYBE_UAVTALK_SLOW5,
-	MSP_MAYBE_UAVTALK_SLOW6
 } msp_state;
 
+typedef enum __attribute__ ((__packed__)) {
+	PROTOCOL_SIMONK = 0,
+	PROTOCOL_BLHELI = 1,
+	PROTOCOL_KISS = 2,
+	PROTOCOL_KISSALL = 3,
+	PROTOCOL_CASTLE = 4,
+	PROTOCOL_4WAY = 0xff,
+} msp_esc_protocol;
+
+struct msp_cmddata_escserial {
+	msp_esc_protocol protocol;
```

Comment:
OK, I will change.  Note it's likely to go back, though, if we take BetaFlight's implementation which does:

```
typedef enum {
    PROTOCOL_SIMONK = 0,
    PROTOCOL_BLHELI = 1,
    PROTOCOL_KISS = 2,
    PROTOCOL_KISSALL = 3,
    PROTOCOL_CASTLE = 4,
    PROTOCOL_COUNT
} escProtocol_e;
```

it will go back.